examples/backendclient/main.go

```diff
@@ -73,57 +99,124 @@ func main() {
 
 	jsonProfile := buffer.String()
 	pbProfile := &pb.MatchObject{}
-	/*
-		err = jsonpb.UnmarshalString(jsonProfile, pbProfile)
-		if err != nil {
-			log.Println(err)
-		}
-	*/
 	pbProfile.Properties = jsonProfile
 
-	log.Println("Requesting matches that fit profile:")
-	ppJSON(jsonProfile)
-	//jsonProfile := bytesToString(jsonData)
-
 	// Connect gRPC client
-	ip, err := net.LookupHost("om-backendapi")
+	ip, err := net.LookupHost(beHost)
```

Comment:
merge the 2 flags dont do lookup just have grpc do it

Reply:
or string concat

Reply:
Removed both - for now it is ok to just hardcode to the existing service names / ports.